pages.id/common/nvm.md

```diff
@@ -0,0 +1,33 @@
+# nvm
+
+> Memasang, melepas, atau mengganti versi Node.js yang dipakai.
```

Comment:
gTranslate says `mengganti` is more similar to `replace` in English. Switch in the main page is used to say that you can go to another node.js version and go back when you want to use another. I just want to check if this is the best verb in Indonesian.

Reply:
It's possible to replace `mengganti` to `mengalihkan` which has a more similar meaning to `switch`. However, the phrase `mengganti versi Node.js yang dipakai` is more understandable towards Indonesian users yet still have the closest meaning of `replace the currently used Node.js version` or `use a specific version of Node.js in the current shell` (as in the description of `nvm use node_version`).

I also didn't use the phrase `in the current shell` in the translation since it's difficult to translate `(computer) shell` in Indonesian. The closest word for `shell` is [`cangkang`](https://kbbi.kemdikbud.go.id/entri/cangkang), which means the `shell` as in animal's body (e.g. molluscs). Some people do translate this term into `syel`, which is [**not** a standard Indonesian word](https://kbbi.kemdikbud.go.id/entri/syel) at all.

So my strategy for translating this page to Indonesian is by telling them that `nvm` allows users to replace the currently-used Node.js version, as in current shell session.